src/net/sourceforge/kolmafia/MonsterData.java

```diff
@@ -1858,6 +1860,31 @@ private String makeItemCount(MonsterDrop drop) {
     return drop.itemCount().isEmpty() ? "" : drop.itemCount() + " ";
   }
 
+  public String getFact() {
+    if (!(KoLCharacter.hasSkill(SkillPool.JUST_THE_FACTS)
+        && StandardRequest.isAllowed(RestrictedItemType.SKILLS, "Just the Facts"))) {
+      return null;
+    }
+    StringBuffer buffer = new StringBuffer();
```

Comment:
there are such a low number of concatentations in this method that not using a buffer at all is likely better. Just return `"<br />Just the Facts:" + f`.

Reply:
Done